chromium_src/components/error_page/common/localized_error.cc

```diff
@@ -0,0 +1,33 @@
+#include "components/url_formatter/url_formatter.h"
+#include "net/base/escape.h"
+#include "url/gurl.h"
+
+namespace brave_error_page {
+
+// Hardcode these constants to avoid dependencies on //chrome and //content.
+// Note: This is following what has been done in original source of this file.
+// unfortunately, the original kChromeUIScheme variable is in an anonymous
+// namespace.
+const char kChromeUIScheme[] = "chrome";
+const char kBraveUIScheme[] = "brave";
+
+std::u16string GetFailedUrlString(GURL failed_url) {
+  if (failed_url.scheme() == kChromeUIScheme) {
+    GURL::Replacements replacements;
+    replacements.SetSchemeStr(kBraveUIScheme);
+
+    failed_url = failed_url.ReplaceComponents(replacements);
+  }
+
+  return url_formatter::FormatUrl(
+      failed_url, url_formatter::kFormatUrlOmitNothing,
+      net::UnescapeRule::NORMAL, nullptr, nullptr, nullptr);
+}
+}  // namespace brave_error_page
+
+#define failed_url_string(FORMATTED_URL) \
+  failed_url_string = brave_error_page::GetFailedUrlString(failed_url);
+
+#include "../../../../components/error_page/common/localized_error.cc"
```

Comment:
pls use `"src/components/...` prefix

Reply:
👍 I was cargo-culting https://github.com/brave/brave-core/blob/5293f0cab08816819bb307d02e404c2061e4368d/chromium_src/chrome/browser/browser_about_handler.cc but that's much nicer.